scripts/start-multi.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Requirements:
+# - nginx installed
+# - temporal server start-dev
+
+set -euo pipefail
+
+WD="${WD-tmp/multi}"
+
+rm -fR "${WD}" && mkdir -p "${WD}"
+
+./bin/ak server setup -m dev
+
+akup() {
+    port="$1"
+    ./bin/ak up -m dev \
+        -c "temporalclient.start_dev_server_if_not_up=false" \
+        -c "http.addr=0.0.0.0:${port}" \
+        -c "logger.zap.outputPaths=${WD}/${port}.log" \
+        >& "${WD}/${port}.out" &
+    echo $!
+}
+
+p1=$(akup 9981)
+p2=$(akup 9982)
+p3=$(akup 9983)
+
+trap 'kill $p1 $p2 $p3' 0
```

Comment:
When running and the hitting `CTRL-C` I get:

```
./scripts/start-multi.sh: line 1: kill: (199474) - No such process
./scripts/start-multi.sh: line 1: kill: (199476) - No such process
./scripts/start-multi.sh: line 1: kill: (199478) - No such process
```

Reply:
I think we shouldn't reinvent the wheel and manage subprocesses, load balancer and such.
docker compose?